homework/kaydunov_denis/shopping_list_application_part4/src/main/java/github/kaydunovdenis/bean/ShoppingCart.java

```diff
@@ -0,0 +1,62 @@
+package github.kaydunovdenis.bean;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class ShoppingCart {
+    private String name;
+    private List<Product> productList;
```

Comment:
should it be static?

Reply:
No. I think that each ShoppingCart should have a separate personalized list of products.